### PR TITLE
Use alphabetical, rather than numeric, label for column

### PR DESCRIPTION
**Pull request by lokal-profil**
Bug: [T242112](https://phabricator.wikimedia.org/T242112)

### CODE REVIEW

**Review by sebastian-berlin-wmse — CHANGES_REQUESTED** [review]
Looks good. Just need to be rebased and there is a typo in the commit message: "then" should be "than".

File: project_start.py [hunk 1]
```diff
@@ -71,8 +71,10 @@ def read_goals(tsv, settings):
         fulfillment = row[1]
         if fulfillment:
             fulfillments[name] = fulfillment
+        # Convert alphabetic label of column to numeric one where A = 0
+        first_project_column = ord(config["goals"]["first_project_column"]) - 65  # noqa: E501
```
Comment (sebastian-berlin-wmse) [inline project_start.py]: Replace `config["goals"]` with `settings`.

**Review by sebastian-berlin-wmse — DISMISSED** [review]
One tiny thing left.

File: project_start.py (hunk 1 above)
Comment (lokal-profil) [reply project_start.py]: Done

**Review by sebastian-berlin-wmse — APPROVED** [review]

**Pull request merged**